Sources/ArcGISToolkit/Components/FormView/FormInputs/ComboBoxInput.swift

```diff
@@ -24,9 +24,12 @@ struct ComboBoxInput: View {
     /// The model for the ancestral form view.
     @EnvironmentObject var model: FormViewModel
     
-    /// The model for the input.
-    @StateObject var inputModel: FormInputModel
-    
+    /// State properties for element events.
```

Comment:
You don't want this to be a doc comment because it applies to more than just the following line. When a comment applies to more than one line, I recommend putting a blank line after it so that it is clear that it doesn't go with just the following line. Also, state properties should be private:

```swift
// State properties for element events.

@State private var isRequired: Bool = false
@State private var isEditable: Bool = false
@State private var value: Any?
@State private var formattedValue: String = ""
```